tests/core-addon/selenium/core_addon.js

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const { Builder, By, until } = require("selenium-webdriver");
+const firefox = require("selenium-webdriver/firefox");
+
+const firefoxOptions = new firefox.Options();
+firefoxOptions.setPreference("xpinstall.signatures.required", false);
+
+// Unset this to run the UI (useful for local testing).
+firefoxOptions.headless();
+// This is the path to Firefox Nightly on Ubuntu with the Mozilla PPA
+if (process.platform === "linux") {
+  firefoxOptions.setBinary("/usr/bin/firefox-trunk");
+}
+const driver = new Builder()
+  .forBrowser("firefox")
+  .setFirefoxOptions(firefoxOptions)
+  .build();
+
+(async function install() {

```

Comment:
We want to use the Mocha runner for tests. Can you reshape this to to work with Mocha? Here's an example: https://github.com/mochajs/mocha-examples/blob/master/packages/selenium/test/selenium.spec.js